client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -1,39 +1,54 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
+import React, { PropTypes } from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
+import { getSelectedSiteId } from 'state/ui/selectors';
+
 import { getCurrentlyEditingProduct } from '../../state/ui/products/selectors';
+import { getProductCategories } from '../../state/wc-api/product-categories/selectors';
 import { editProduct, editProductAttribute } from '../../state/ui/products/actions';
+import { getProductCategories as fetchProductCategories } from '../../state/wc-api/product-categories/actions';
```

Comment:
For sanity's sake, how about we change the action to `fetchProductCategories` too? :)